TeamCode/src/main/java/org/firstinspires/ftc/teamcode/FourMotorChasis.java

```diff
@@ -0,0 +1,74 @@
+package org.firstinspires.ftc.teamcode;
+
+import com.qualcomm.robotcore.eventloop.opmode.TeleOp;
+import com.qualcomm.robotcore.hardware.DcMotor;
+import com.qualcomm.robotcore.util.Range;
+import com.qualcomm.robotcore.eventloop.opmode.OpMode;
+
+/**
+ * Created by pston on 10/15/2017
+ */
+
+@TeleOp(name="Four Motor Test", group="Test")
```

Comment:
#Fix Can you change this name to something more descriptive, like "DriveTrain"? It's under the Test group so you can leave test off the name.